neqo-transport/src/recovery.rs

```diff
@@ -494,9 +505,18 @@ impl LossRecovery {
         qinfo!(
             [self],
             "sent packets {} {} {}",
-            self.spaces[PNSpace::Initial].sent_packets.len(),
-            self.spaces[PNSpace::Handshake].sent_packets.len(),
-            self.spaces[PNSpace::ApplicationData].sent_packets.len()
+            self.spaces[PNSpace::Initial]
+                .sent_packets
+                .values()
+                .any(|sp| sp.ack_eliciting),
+            self.spaces[PNSpace::Handshake]
+                .sent_packets
+                .values()
+                .any(|sp| sp.ack_eliciting),
+            self.spaces[PNSpace::ApplicationData]
+                .sent_packets
+                .values()
+                .any(|sp| sp.ack_eliciting)
```

Comment:
how about outputting both number of packets AND if any are ack-eliciting in this qinfo?